### PR TITLE
New check 7163 Secrets Manager key rotation enabled @Daniel-Peladeau

### PR DESCRIPTION
**Pull request by Daniel-Peladeau**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by Daniel-Peladeau** [comment]
Check if Secrets Manager key rotation enabled.

File: checks/check_extra7163 [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#       --secret-id <value>
+#       [--client-request-token <value>]
+#       [--rotation-lambda-arn <value>]
+#       [--rotation-rules <value>]
+#       [--cli-input-json <value>]
+#       [--generate-cli-skeleton <value>]
+
+
+CHECK_ID_extra7163="7.163"
+CHECK_TITLE_extra7163="[extra7163] Check if Secrets Manager key rotation enabled"
```
Comment (toniblyx) [inline checks/check_extra7163]: "Check if Secrets Manager key rotation is enabled" - Add `is` please.

File: checks/check_extra7163 [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        for secret in $LIST_OF_SECRETS; do
+          KEY_ROTATION_ENABLED=$($AWSCLI secretsmanager describe-secret $PROFILE_OPT --region $regx --secret-id $secret | jq '.RotationEnabled')
+          if [[ $KEY_ROTATION_ENABLED == true ]]; then
+            textPass "$regx: $secret has key rotation enabled." "$regx" "$secret"
+          else
+            textFail "$regx: $secret does not have key rotation enabled." "$regx" "$secret"
+          fi
+        done
+      else
+        textPass "$regx: No SecretsManager secrets found." "$regx"
```
Comment (toniblyx) [inline checks/check_extra7163]: `Secrets Manager` with space in between, isn't it?

File: checks/check_extra7163 [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+CHECK_DOC_extra7163="https://docs.aws.amazon.com/secretsmanager/latest/userguide/rotating-secrets_strategies.html"
+CHECK_CAF_EPIC_extra7163="Data Protection"
+
+extra7163(){
+    # "Check if Secrets Manager key rotation enabled"
+    for regx in $REGIONS; do
+      LIST_OF_SECRETS=$($AWSCLI secretsmanager list-secrets $PROFILE_OPT --region $regx --query 'SecretList[*].Name' --output text)
+      if [[ $LIST_OF_SECRETS ]]; then
+        for secret in $LIST_OF_SECRETS; do
+          KEY_ROTATION_ENABLED=$($AWSCLI secretsmanager describe-secret $PROFILE_OPT --region $regx --secret-id $secret | jq '.RotationEnabled')
```
Comment (toniblyx) [inline checks/check_extra7163]: if using jq, please force the output to json adding `--output json`

**Comment by toniblyx** [comment]
Thanks @Daniel-Peladeau, I have added some comments for you to review.

**Review by toniblyx — CHANGES_REQUESTED** [review]

**Comment by Daniel-Peladeau** [comment]
New commit just added with the three requested changes.

**Comment by toniblyx** [comment]
Thanks!

**Pull request merged**